File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/ClientClassificationModel.ts
import { ClientModel } from "./clientModel";

export class ClientClassificationModel {
    clientClassificationId: number;
    clientClassificationCode: string;

    clientClassificationNameEn: string;
    clientClassificationNameAr: string;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    icon: string;
    color: string;
    displayOrder: number | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

    notes:string;

}

File: SofiForce.Models/Models/EntityModels/ClientClassificationModel.cs
using System;

namespace Models
{
    public class ClientClassificationModel
    {
        public int ClientClassificationId { get; set; }
        public string ClientClassificationCode { get; set; }

        public string ClientClassificationNameEn { get; set; }
        public string ClientClassificationNameAr { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public int? DisplayOrder { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
        public string Notes { get; set; }
    }
}
